### PR TITLE
fix(components): fixed close/open modal function - CAX-829

### PR DESCRIPTION
**Pull request by helen-v**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/modal/Modal.ts [hunk 1]
```diff
@@ -163,7 +163,7 @@ export class Modal extends FocusTrapMixin(LitElement) {
     this.deactivateFocusTrap!();
 
     setTimeout(() => {
-      this.closeModal();
```
Comment (futurethang) [inline web-components/src/components/modal/Modal.ts:166]: This appears to be the only location where `private closeModal()` is used, so the function can be deleted from the file as well from line 112

**Review by futurethang — CHANGES_REQUESTED** [review]

File: web-components/src/components/modal/Modal.ts (hunk 1 above)
Comment (helen-v) [reply web-components/src/components/modal/Modal.ts:166]: Agree! Deleted.

File: web-components/src/components/modal/Modal.test.ts [hunk 2]
```diff
@@ -216,6 +214,5 @@ describe("Modal Component", () => {
     jest.runAllTimers();
 
     expect(document.activeElement).not.toEqual(input);
```
Comment (futurethang) [inline web-components/src/components/modal/Modal.test.ts:216]: These tests might need rethinking. `document.activeElement()` cannot penetrate the shadowDom, and will always return the outermost custom element, and so this test assertion will always return true.  You'd either need to test in a different way, or replace the native `activeElement` with something that can recursively search into shadowDoms.

**Review by futurethang — CHANGES_REQUESTED** [review]

File: web-components/src/components/modal/Modal.test.ts (hunk 2 above)
Comment (helen-v) [reply web-components/src/components/modal/Modal.test.ts:216]: Kevin, I agree. And when Kostay will available I'll ask him about logic, Or rewrite tests for modal component.

**Review by yananym — APPROVED** [review]

**Review by yananym — APPROVED** [review]

**Pull request merged**